civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -83,3 +81,13 @@ $row-margin-bottom: 15px;
   margin-top: 7px;
   display: inline-block;
 }
+
+.chr_leave-request-modal__disabled {
```

Comment:
This disabled class might be useful in other situation too. So please create a generic class under `sites/all/themes/civihr_employee_portal_theme/civihr_default_theme/scss/base/_helpers.scss`.